addons/more-links/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "More links",
-  "description": "Adds links for URLs pointing outside scratch.mit.edu.",
+  "name": "More clickable links",
+  "description": "Makes URLs pointing outside scratch.mit.edu clickable.",
```

Comment:
I suppose links are clickable in the first place?

Reply:
Yeah, "clickable" part is redundant.